modules/crafting/server.lua

```diff
@@ -112,7 +112,9 @@ lib.callback.register('ox_inventory:craftItem', function(source, id, index, reci
 			end
 
 			local craftedItem = Items(recipe.name)
-			local newWeight = left.weight + (craftedItem.weight + (recipe.metadata?.weight or 0)) * (recipe.count or 1)
+			math.randomseed(os.time() + source)
+			local craftCount = (type(recipe.count) == 'number' and recipe.count) or (type(recipe.count) == 'table' and table.type(recipe.count) == 'array' and math.random(recipe.count[1], recipe.count[2])) or 1
```

Comment:
You can just check table.type is an array without checking the variable is a table.